active/operators/trashman/check_brat_completeness.py

```diff
@@ -51,24 +54,24 @@ def write_run_details(run_id, check_date, brat_files, stale_threshold=common_var
     tgt_insert_stmt = ("INSERT INTO {job_table}"
                        "({run_id}, "
                        "stale_threshold_days,"
-                       " stale_check_date,"
+                       " job_start_date,"
                        " directory_location,"
                        " brat_last_modified_date,"
                        " job_status,"
                        " {brat_id},"
                        " hdcorcablobid,"
                        " hdcpupdatedate) "
-                       "VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s)".format(job_table=common_variables.AF4_COMPLETE_BRAT_TABLE,
+                       "VALUES (%s, %s, %s, %s, %s, %s, %s, %s)".format(job_table=common_variables.AF4_COMPLETE_BRAT_TABLE,
```

Comment:
I count 9 values to insert...stale_check_date was removed, but job_start_date was added, so still 9 right?

Reply:
you're right!